controls/os_spec.rb

```diff
@@ -282,3 +282,25 @@
     end
   end
 end
+
+control 'os-14' do
+  impact 1.0
+  title 'Check pam config - RedHat specific'
+  desc 'Check config of files in /etc/pam.d.'
+
+  pam_files = ['/etc/pam.d/system-auth', '/etc/pam.d/password-auth']
+
+  pam_files.each do |pam_file|
+    describe file(pam_file) do
+      it { should exist }
+      it { should be_owned_by 'root' }
+      its('group') { should eq 'root' }
+      its(:content) { should match /^password    requisite     pam_pwhistory.so remember=60 use_authtok$/ }
```

Comment:
This collects 60 passwords. I suggest to go with the CIS default of 5 and make this the default argument for this policy. You can than override the policy with 60 for your needs.

What do you think if we also split this into multiple pieces as CIS DIL 5.3 is doing?

Reply:
I set the default back to 5